nfs/src/rpc_task.cpp

```diff
@@ -1149,7 +1149,29 @@ void rpc_task::run_read()
              * after it acquires the lock.
              */
             found_in_cache = false;
-            read_from_server(bc_vec[i]);
+
+            /*
+             * TODO: If we have just 1 bytes chunk to fill, which is the most common case,
+             *       avoid creating child task and process evrything in this same task.
+             */
+
+            /*
+             * Create a child rpc task to issue the read RPC to the backend.
+             */
+            struct rpc_task *child_tsk =
+                get_client()->get_rpc_task_helper()->alloc_rpc_task(FUSE_READ);
+
+            child_tsk->init_read(
+                rpc_api->req,
+                rpc_api->read_task.get_ino(),
+                rpc_api->read_task.get_offset(),
+                rpc_api->read_task.get_size(),
+                nullptr /* fileinfo */);
```

Comment:
let's not create asymmetry with the actual read. Though we don't use fileinfo, either we remove it from both places or add it here. For now I'll add it here